ubuntu/README.md

```diff
@@ -39,6 +39,12 @@ packer build -var customize_script=my-changes.sh -var ubuntu_series=jammy \
 the VM. For example, you can install packages using `apt-get`, call out to
 ansible, or whatever you want.
 
+Building using make:
+
+```shell
+sudo make custom-cloudimg.tar.gz SERIES=jammy
```

Comment:
any reason for needing `sudo`? we moved to fuseFS to avoid this

Reply:
A bad habit! Let's remove that